### PR TITLE
Acrs 15 save and exit

### PR DESCRIPTION
**Pull request by TemitopeAyokuHO**
## What? 
[ACRS-15](https://collaboration.homeoffice.gov.uk/jira/browse/ACRS-15) Save and Exit (returning user feature)

## Why? 
The save and exit functionality exists on several pages throughout the form. it helps user and save the current form and return in future to continue 

## How? 
- Added save and exit behaviour functionality to allow user to Save and Exit form when user decides to continue later in 
save-and-exit.js file
- Merged ACRS- 61 and ACRS-35 branch to display and test if they work ok 
- Added html pages to inform user the details has been saved

## Testing?
Tested on my local machine 

## Check list

- [x] I have reviewed my own pull request for linting issues (e.g. adding new lines)
- [ ] I have written tests (if relevant)
- [x] I have created a JIRA number for my branch
- [x] I have created a JIRA number for my commit
- [x] I have followed the chris beams method for my commit https://cbea.ms/git-commit/
here is an [example commit](https://github.com/UKHomeOfficeForms/hof/commit/810959f391187c7c4af6db262bcd143b50093a6e)
- [x] Ensure drone builds are green especially tests
- [x] I will squash the commits before merging

### CODE REVIEW

File: apps/acrs/index.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ module.exports = {
       behaviours: [SummaryPageBehaviour, CheckInformationGivenBehaviour],
       sections: require('./sections/summary-data-sections'),
       backLink: false,
-      journeyStart: '/who-is-completing-form'
+      journeyStart: '/who-completing-form'
```
Comment (jamiecarterHO) [inline apps/acrs/index.js:31]: I think it needs to be `next: '/who-completing-form'` else it won't move on from this page

**Review by jamiecarterHO — CHANGES_REQUESTED** [review]
One little change

**Comment by TemitopeAyokuHO** [comment]
> One little change

yes

**Review by jamiecarterHO — APPROVED** [review]

File: apps/acrs/index.js (hunk 1 above)
Comment (TemitopeAyokuHO) [reply apps/acrs/index.js:31]: It navigate and working fine with journeyStart. If you change it to next, you will need to modify continue-report.js which will break most of the code

Comment (jamiecarterHO) [reply apps/acrs/index.js:31]: Yeah all good. Have approved

**Pull request merged**